### PR TITLE
Adds basic actor reminder integration tests

### PR DESCRIPTION
**Pull request by JoshVanL**

### CODE REVIEW

File: tests/integration/suite/actors/reminders/basic.go [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+	require.NoError(t, err)
+
+	require.EventuallyWithT(t, func(c *assert.CollectT) {
+		resp, rErr := client.Do(req)
+		require.NoError(c, rErr)
+		assert.NoError(c, resp.Body.Close())
+		assert.Equal(c, http.StatusOK, resp.StatusCode)
+	}, time.Second*10, time.Millisecond*100, "actor not ready in time")
+
+	body := `{"dueTime": "100ms"}`
```
Comment (ItalyPaleAle) [inline tests/integration/suite/actors/reminders/basic.go:106]: Consider setting `dueTime` to `0` so the reminder is always executed right away without concerns

**Review by ItalyPaleAle — APPROVED** [review]

**Review by ItalyPaleAle — APPROVED** [review]

**Review by daixiang0 — APPROVED** [review]
LGTM

File: tests/integration/suite/actors/reminders/basic.go [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+func (b *basic) Run(t *testing.T, ctx context.Context) {
+	b.place.WaitUntilRunning(t, ctx)
+	b.daprd.WaitUntilRunning(t, ctx)
+
+	client := util.HTTPClient(t)
+
+	daprdURL := "http://localhost:" + strconv.Itoa(b.daprd.HTTPPort())
+
+	req, err := http.NewRequest(http.MethodPost, daprdURL+"/v1.0/actors/myactortype/myactorid/method/foo", nil)
```
Comment (mukundansundar) [inline tests/integration/suite/actors/reminders/basic.go:96]: probably add in NewRequestWithContext?

File: tests/integration/suite/actors/reminders/basic.go [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+	require.EventuallyWithT(t, func(c *assert.CollectT) {
+		resp, rErr := client.Do(req)
+		require.NoError(c, rErr)
+		assert.NoError(c, resp.Body.Close())
+		assert.Equal(c, http.StatusOK, resp.StatusCode)
+	}, time.Second*10, time.Millisecond*100, "actor not ready in time")
+
+	body := `{"dueTime": "0ms"}`
+	req, err = http.NewRequest(http.MethodPost, daprdURL+"/v1.0/actors/myactortype/myactorid/reminders/remindermethod", strings.NewReader(body))
```
Comment (mukundansundar) [inline tests/integration/suite/actors/reminders/basic.go:107]: preferably template strings please ...

**Review by mukundansundar — APPROVED** [review]
probably we can add a basic timer test if not already there ...

File: tests/integration/suite/actors/reminders/basic.go (hunk 2 above)
Comment (JoshVanL) [reply tests/integration/suite/actors/reminders/basic.go:96]: Done :+1:

File: tests/integration/suite/actors/reminders/basic.go (hunk 3 above)
Comment (JoshVanL) [reply tests/integration/suite/actors/reminders/basic.go:107]: What do you mean by "template" the string here?

Comment (mukundansundar) [reply tests/integration/suite/actors/reminders/basic.go]: the URLs are all mostly the same form with small changes ... that can use a template with fmt.Sprintf() formatter template...

Comment (JoshVanL) [reply tests/integration/suite/actors/reminders/basic.go]: Gothcha- done!

**Review by mukundansundar — APPROVED** [review]
lgtm

**Review by artursouza — APPROVED** [review]

**Pull request merged**